en/flight_controller/raspberry_pi_navio2.md

```diff
@@ -176,87 +68,61 @@ Make sure the Pi has network access, and you can SSH into it.
 
 ```sh
 echo "Hello" > hello.txt
-scp hello.txt pi@px4autopilot:/home/pi/
+scp hello.txt pi@navio.local:/home/pi/
 rm hello.txt
 ```
 
-This should copy over a "hello.txt" file into the home folder of your RPi.
+This should copy over a "hello.txt" file into the home folder of your Pi.
 Validate that the file was indeed copied, and you can proceed to the next step.
 
 ## Building the Code
 
-Either build the source code on your development computer ("cross-compiler" build) or build it on the RaPi ("native" build) as shown below.
+Follow the instructions below to build the source code on your developmenet machine (Ubuntu 18.04) and transfer the compiled program to the Pi, unfortunetealy, the option to build the code natively(on the Pi) is no longer available.
```

Comment:
Does it have to be Ubuntu 18.04? Or is this just what you used? Would these work on Ubuntu 22.04. I ask because Ubuntu 18.04 is EOL.

Reply:
Correct, it must be 18.04 or older, I was skeptical at first too, but apparently the binary compiled by any newer version of Ubuntu is too new for the Pi, and the compiler scripts are only made for Ubuntu.  I tried on 22.04 as I already had a VM setup, but ran into some weird errors on the Pi (error: static assertion failed: atomic is not lock-free for the given type T).

Reply:
Thanks. I'm going to call this out in a note.